src/frontend/js/types/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Nullable } from 'types/utils';
 
 export enum CourseStateTextEnum {
-  CLOSING_ON = 'closing on',
+  CLOSING_ON = 'open for enrollment',
```

Comment:
```suggestion
  ENROLLMENT_OPENED = 'open for enrollment',
```

Reply:
I have a doubt to, but you give me a confirmation, thanks. =)